res/values/arrays.xml

```diff
@@ -2,76 +2,76 @@
-    <item>bone</item>
-    <item>jcuken_ua</item>
-    <item>bangla</item>
-    <item>devanagari_1</item>
-    <item>devanagari_2</item>
-    <item>he_il_1452_1</item>
-    <item>he_il_1452_2</item>
-    <item>ar_pc</item>
-    <item>ar_alt</item>
-    <item>persian</item>
+    <item>latn_qwerty_us</item>
+    <item>latn_colemak</item>
+    <item>latn_dvorak</item>
+    <item>arab_ibm</item>
+    <item>arab_ibm_ma</item>
+    <item>arab_ir</item>
+    <item>beng_rh</item>
+    <item>cyrl_jcuken_ru</item>
+    <item>cyrl_jcuken_uk</item>
+    <item>cyrl_yaverti</item>
+    <item>deva_inscript</item>
+    <item>deva_lv</item>
+    <item>grek_qwerty</item>
+    <item>hang_dubeolsik_kr</item>
+    <item>hebr_</item>
+    <item>hebr_2</item>
+    <item>latn_azerty_fr</item>
+    <item>latn_bone</item>
+    <item>latn_neo2</item>
+    <item>latn_qwerty</item>
+    <item>latn_qwerty_br</item>
+    <item>latn_qwerty_es</item>
+    <item>latn_qwerty_lv</item>
+    <item>latn_qwerty_no</item>
+    <item>latn_qwerty_pl</item>
+    <item>latn_qwerty_se</item>
+    <item>latn_qwerty_tr</item>
+    <item>latn_qwerty_vi</item>
+    <item>latn_qwertz</item>
+    <item>latn_qwertz_cz</item>
+    <item>latn_qwertz_de</item>
+    <item>latn_qwertz_hu</item>
+    <item>latn_qwertz_sk</item>
     <item>custom</item>
   </string-array>
   <string-array name="pref_layout_entries">
     <item>None</item>
-    <item>AZERTY</item>
-    <item>QWERTY</item>
-    <item>QWERTY (Brasileiro)</item>
-    <item>QWERTY (Español)</item>
-    <item>QWERTY (Greek)</item>
-    <item>QWERTY (Korean)</item>
-    <item>QWERTY (Latvian)</item>
-    <item>QWERTY (Norwegian)</item>
-    <item>QWERTY (Swedish)</item>
-    <item>QWERTY (Hungarian)</item>
-    <item>QWERTY (Türkçe)</item>
-    <item>QWERTY (Polski)</item>
-    <item>QWERTY (Vietnamese)</item>
-    <item>ЙЦУКЕН (Русский)</item>
-    <item>QWERTZ</item>
-    <item>QWERTZ (Czech)</item>
-    <item>QWERTZ (Deutsch)</item>
-    <item>QWERTZ (Hungarian)</item>
-    <item>QWERTZ (Slovak)</item>
-    <item>Bulgarian (Phonetic Traditional)</item>
-    <item>Dvorak</item>
+    <item>QWERTY [United States of America]</item>
     <item>Colemak</item>
-    <item>Neo 2</item>
-    <item>Bone</item>
-    <item>ЙЦУКЕН (Українська)</item>
-    <item>বাংলা</item>
-    <item>देवनागरी (हिंदी)-1</item>
-    <item>देवनागरी (हिंदी)-2</item>
-    <item>Hebrew SI-1452-1</item>
-    <item>Hebrew SI-1452-2</item>
-    <item>Arabic PC</item>
-    <item>Arabic ALT</item>
+    <item>Dvorak</item>
+    <item>IBM Arabic</item>
+    <item>IBM Arabic (Mostafa Ayesh)</item>
     <item>Persian</item>
+    <item>Bengali (Md Rasel Hossain)</item>
+    <item>ЙЦУКЕН [Russia]</item>
+    <item>ЙЦУКЕН [Ukraine]</item>
+    <item>ЯВЕРТЪ</item>
+    <item>InScript Devanagari</item>
+    <item>Devanagari (Luke Videckis)</item>
+    <item>QWERTY [Hellas]</item>
+    <item>Hebrew</item>
+    <item>Hebrew 2</item>
+    <item>두벌식 [Republic of Korea]</item>
```

Comment:
I don't like the inconsistently translated country names. Why is that ? It seems that you already made all the translations in the previous PR.

Reply:
Which one did you think is inconsistent? The naming rule is to follow the shortened form of the formal name and if there are more than one country/territory having the same shortened form or causing confusions with a geographical term, use full name.
Here, when “United States of America” is written as “USA”, “Republic of Korea” needs to be renamed “Ro Korea”.

Reply:
Some of the names are spelled in english while other are not.

I'd also prefer smaller names.

> or causing confusions with a geographical term

I don't care with this one. I want layouts to have slightly different names, that's it.

Reply:
Could you explain what that “spelled in english” mean? Did you mean name not abbreviated?
If you don't care abou that geographical term, then “United States” could be called “America” instead and cause confusion with the Latin American variaet.

Reply:
This is way too complicated. "QWERTY (US)" is the most common name.

Reply:
“Hellas” is an English word. It is the native name of Greece and is the noun form of “Hellenic” in the formal name of Greece “Hellenic Republic”.

Reply:
I liked translated layout names and countries better (though it was language names before, which I don't prefer). I think this might be what non-english users expect.